src/borrow_tracker/tree_borrows/unimap.rs

```diff
@@ -207,6 +207,12 @@ impl<V> UniValMap<V> {
         mem::swap(&mut res, &mut self.data[idx.idx.to_usize()]);
         res
     }
+
+    /// Weather the datastructure has been allocated.
+    /// Does not mean the map contains any values.
+    pub fn is_initialized(&self) -> bool {
```

Comment:
This exposes internal state in an odd way -- can't we have an `is_empty` instead?